packages/edit-site/src/components/header-edit-mode/document-actions/style.scss

```diff
@@ -58,15 +58,19 @@
 	}
 }
 
-.edit-site-document-actions__shortcut {
-	color: $gray-700;
-	text-align: right;
+.edit-site-document-actions__shortcut,
+.edit-site-document-actions__back {
+	color: darken($gray-700, 3%); // Darken the secondary color to meet requirements against a gray background.
```

Comment:
Could we use `gray-800` rather than a 'custom' color? 🤔

Reply:
We can, yes, though it's slightly dark innit? That was the intent by just darkening a little bit.

Reply:
For me it's worth that potential trade-off to avoid deviating from the scale. I don't actually mind the darkness:

<img width="567" alt="Screenshot 2023-06-13 at 11 38 11" src="https://github.com/WordPress/gutenberg/assets/846565/62dc793a-7bde-4d58-929a-461aebb672c3">

It's just a nitpick though :)

Reply:
Done!